src/pages/ErrorPage.tsx

```diff
@@ -0,0 +1,5 @@
+export const ErrorPage = () => {
```

Comment:
Doesn't match the task rewuirement
![image](https://github.com/mate-academy/react_people-table-basics/assets/99403198/81bc3f19-18c5-4694-b1f0-30004018925c)
```suggestion
export const NotFoundPage = () => {
```